server/traditionalTable/mappings/panEuropean/index.js

```diff
@@ -1,5 +1,6 @@
 const table11a = require('./table1_1a')
 
 module.exports = {
-  panEuropean11a: table11a,
+  // panEuropean11a: table11a,
+  table_1_1a: table11a,
```

Comment:
Could also be:

```
const table_1_1a = require('./table1_1a')
module.exports = {
  table_1_1a,
}
```

Reply:
yes that would be better

Reply:
But eslint will htrow an error for this - so we can keep it as is for now